packages/contracts/src/plugins/governance/multisig/release-metadata.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Multisig",
+  "name": "",
```

Comment:
why do you remove the name?

Reply:
Fixed in 5e13df205a68d9231bca8c6cf1b57d4d614d4480